packages/codemod/src/transforms/rename-mod-css-to-module-css.ts

```diff
@@ -0,0 +1,53 @@
+import fs from 'fs';
+import path from 'path';
+import type { API, ASTPath, FileInfo, ImportDeclaration } from 'jscodeshift';
+import type { TRunnerOptions } from '../types';
+
+function renameModCssToModuleCss(
+  file: FileInfo,
+  api: API,
+  options: TRunnerOptions
+) {
+  const j = api.jscodeshift;
+
+  let hasModifications = false;
+
+  function renameImportModCssToModuleCss(astPath: ASTPath<ImportDeclaration>) {
+    const node = astPath.node;
+
+    if (
+      typeof node.source.value === 'string' &&
+      node.source.value.endsWith('.mod.css')
+    ) {
+      if (options.dry) {
+        api.stats(file.path);
+      } else {
+        const previousCssModulesFilePath = node.source.value;
+        const nextCssModulesFilePath = previousCssModulesFilePath.replace(
+          '.mod.css',
+          '.module.css'
+        );
+        node.source.value = nextCssModulesFilePath;
+        hasModifications = true;
+
+        fs.renameSync(
+          path.join(path.dirname(file.path), previousCssModulesFilePath),
+          path.join(path.dirname(file.path), nextCssModulesFilePath)
+        );
```

Comment:
Side effect of the transform (which updates the import) to also rename the actual CSS module file.